verify.go

```diff
@@ -155,15 +155,25 @@ func (v *IDTokenVerifier) Verify(ctx context.Context, rawIDToken string) (*IDTok
 		return nil, fmt.Errorf("oidc: failed to unmarshal claims: %v", err)
 	}
 
+	distributedClaims := make(map[string]claimSource)
+
+	//step through the token to mp claim names to claim sources"
+	for cn, src := range token.ClaimNames {
+		if src != "" {
```

Comment:
probably

```
if src == "" {
    // return error
}
s, ok := token.ClaimsSources[src]
if !ok {
    // return error because source name doesn't exist
}
distributedClaims[cn] = s
```